Failure to find io.openliberty.tools:liberty-maven-app-parent:pom:${project.version}

Using the [liberty-archetype-webapp archtype instructions](https://github.com/OpenLiberty/ci.maven#liberty-archetype-webapp) with the [latest release of 3.3.4](https://repo1.maven.org/maven2/archetype-catalog.xml) doesn't work:

```
kevin@Kevins-MBP helloworld_microprofile $ mvn archetype:generate -DarchetypeGroupId=io.openliberty.tools -DarchetypeArtifactId=liberty-archetype-webapp -DarchetypeVersion=3.3.4 -DgroupId=test -DartifactId=test -Dversion=1.0-SNAPSHOT -B
[INFO] Scanning for projects...
[INFO] 
[INFO] ------------------< org.apache.maven:standalone-pom >-------------------
[INFO] Building Maven Stub Project (No POM) 1
[INFO] --------------------------------[ pom ]---------------------------------
[INFO] 
[INFO] >>> maven-archetype-plugin:3.2.0:generate (default-cli) > generate-sources @ standalone-pom >>>
[INFO] 
[INFO] <<< maven-archetype-plugin:3.2.0:generate (default-cli) < generate-sources @ standalone-pom <<<
[INFO] 
[INFO] 
[INFO] --- maven-archetype-plugin:3.2.0:generate (default-cli) @ standalone-pom ---
[INFO] Generating project in Batch mode
[INFO] Archetype repository not defined. Using the one from [io.openliberty.tools:liberty-archetype-webapp:3.3.5-M1] found in catalog remote
[INFO] ----------------------------------------------------------------------------
[INFO] Using following parameters for creating project from Archetype: liberty-archetype-webapp:3.3.4
[INFO] ----------------------------------------------------------------------------
[INFO] Parameter: groupId, Value: test
[INFO] Parameter: artifactId, Value: test
[INFO] Parameter: version, Value: 1.0-SNAPSHOT
[INFO] Parameter: package, Value: test
[INFO] Parameter: packageInPathFormat, Value: test
[INFO] Parameter: package, Value: test
[INFO] Parameter: version, Value: 1.0-SNAPSHOT
[INFO] Parameter: libertyPluginVersion, Value: ${project.version}
[INFO] Parameter: runtimeVersion, Value: [20.0.0.8,)
[INFO] Parameter: runtimeArtifactId, Value: openliberty-runtime
[INFO] Parameter: groupId, Value: test
[INFO] Parameter: runtimeGroupId, Value: io.openliberty
[INFO] Parameter: artifactId, Value: test
[INFO] Project created from Archetype in dir: /Users/kevin/git/helloworld_microprofile/test
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  2.588 s
[INFO] Finished at: 2021-05-10T12:11:50-07:00
[INFO] ------------------------------------------------------------------------
kevin@Kevins-MBP helloworld_microprofile $ cd test
kevin@Kevins-MBP test $ mvn clean install
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for test:test:1.0-SNAPSHOT: Failure to find io.openliberty.tools:liberty-maven-app-parent:pom:${project.version} in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 8, column 13
 @ 
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]   
[ERROR]   The project test:test:1.0-SNAPSHOT (/Users/kevin/git/helloworld_microprofile/test/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for test:test:1.0-SNAPSHOT: Failure to find io.openliberty.tools:liberty-maven-app-parent:pom:${project.version} in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 8, column 13 -> [Help 2]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
```

--- a/liberty-archetype-ear/pom.xml
+++ b/liberty-archetype-ear/pom.xml
@@ -17,6 +17,22 @@
     </description>
     
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>META-INF/maven/archetype-metadata.xml</include>
+                </includes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>false</filtering>
+                <excludes>
+                    <exclude>META-INF/maven/archetype-metadata.xml</exclude>
+                </excludes>
+            </resource>
+        </resources>
         <extensions>
             <extension>
                 <groupId>org.apache.maven.archetype</groupId>

--- a/liberty-archetype-ear/src/main/resources/META-INF/maven/archetype-metadata.xml
+++ b/liberty-archetype-ear/src/main/resources/META-INF/maven/archetype-metadata.xml
@@ -13,7 +13,7 @@
             <defaultValue>openliberty-runtime</defaultValue>
         </requiredProperty>
         <requiredProperty key="runtimeVersion">
-            <defaultValue>[20.0.0.8,)</defaultValue>
+            <defaultValue>[21.0.0.3,)</defaultValue>
         </requiredProperty>
         <requiredProperty key="libertyPluginVersion">
             <defaultValue>${project.version}</defaultValue>

--- a/liberty-archetype-webapp/pom.xml
+++ b/liberty-archetype-webapp/pom.xml
@@ -13,6 +13,22 @@
     <name>Archetype - liberty-archetype-webapp</name>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>META-INF/maven/archetype-metadata.xml</include>
+                </includes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>false</filtering>
+                <excludes>
+                    <exclude>META-INF/maven/archetype-metadata.xml</exclude>
+                </excludes>
+            </resource>
+        </resources>
         <extensions>
             <extension>
                 <groupId>org.apache.maven.archetype</groupId>

--- a/liberty-archetype-webapp/src/main/resources/META-INF/maven/archetype-metadata.xml
+++ b/liberty-archetype-webapp/src/main/resources/META-INF/maven/archetype-metadata.xml
@@ -13,7 +13,7 @@
             <defaultValue>openliberty-runtime</defaultValue>
         </requiredProperty>
         <requiredProperty key="runtimeVersion">
-            <defaultValue>[20.0.0.8,)</defaultValue>
+            <defaultValue>[21.0.0.3,)</defaultValue>
         </requiredProperty>
         <requiredProperty key="libertyPluginVersion">
             <defaultValue>${project.version}</defaultValue>

--- a/liberty-plugin-archetype/pom.xml
+++ b/liberty-plugin-archetype/pom.xml
@@ -14,6 +14,22 @@
     <packaging>maven-archetype</packaging>
 
     <build>
+        <resources>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>true</filtering>
+                <includes>
+                    <include>META-INF/maven/archetype-metadata.xml</include>
+                </includes>
+            </resource>
+            <resource>
+                <directory>src/main/resources</directory>
+                <filtering>false</filtering>
+                <excludes>
+                    <exclude>META-INF/maven/archetype-metadata.xml</exclude>
+                </excludes>
+            </resource>
+        </resources>
         <extensions>
             <extension>
                 <groupId>org.apache.maven.archetype</groupId>

--- a/liberty-plugin-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml
+++ b/liberty-plugin-archetype/src/main/resources/META-INF/maven/archetype-metadata.xml
@@ -14,7 +14,7 @@
             <defaultValue>openliberty-runtime</defaultValue>
         </requiredProperty>
         <requiredProperty key="runtimeVersion">
-            <defaultValue>[20.0.0.8,)</defaultValue>
+            <defaultValue>[21.0.0.3,)</defaultValue>
         </requiredProperty>
         <requiredProperty key="libertyPluginVersion">
             <defaultValue>${project.version}</defaultValue>

